substrate/frame/revive/src/exec.rs

```diff
@@ -66,6 +66,10 @@ type VarSizedKey = BoundedVec<u8, ConstU32<{ limits::STORAGE_KEY_BYTES }>>;
 
 const FRAME_ALWAYS_EXISTS_ON_INSTANTIATE: &str = "The return value is only `None` if no contract exists at the specified address. This cannot happen on instantiate or delegate; qed";
 
+/// Code hash of existing account without code (keccak256 hash of empty data).
+pub(crate) const EMPTY_CODE_HASH: H256 =
```

Comment:
Not necessary. The `exec` module itself is not public.

```suggestion
pub const EMPTY_CODE_HASH: H256 =
```